docs/source/changelog.rst

```diff
@@ -23,7 +23,7 @@ Bag
 Core
 ++++
 
--
+-  Allow tuples as sharedict keys
```

Comment:
Saw the PR link was missing. Went ahead and added it in PR ( https://github.com/dask/dask/pull/2766 ).